.gitlab/scripts/build_layer.sh

```diff
@@ -42,7 +42,6 @@ docker_build() {
 
     cp $EXTENSION_PATH/datadog_extension.zip $TARGET_DIR/datadog_extension-${FILE_SUFFIX}.zip
     unzip $EXTENSION_PATH/datadog_extension.zip -d $TARGET_DIR/datadog_extension-${FILE_SUFFIX}
-    rm -rf ${EXTENSION_PATH}
```

Comment:
why?

Reply:
previously we didnt remove it, I added this line to only have zips and not dirs in the layers folder, but the private images require this to work properly